docs/pages/access-graph.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Teleport Access Graph
+description: A reference for Teleport Access Graph.
+---
+
+<Admonition type="tip" title="Preview">
+  Teleport Access Graph is currently in Preview. Currently, it is only available
+  to the Teleport Cloud Team users. Currently, only SSH server are supported.
```

Comment:
```suggestion
  Teleport Access Graph is currently in Preview. Currently, it is only available
  to Teleport Team users, and only visualizes access to SSH servers enrolled with your Teleport clusters.
```
Suggested to:
- Make this consistent with the rest of the docs, which say "Teleport Team", rather than "Teleport Cloud Team"
- Be more explicit about what "are supported" means. I'm not sure if my suggestion is technically correct, though.